lavalink/models.py

```diff
@@ -16,16 +17,15 @@ class TrackNotBuilt(Exception):
 
 class AudioTrack:
     __slots__ = ('track', 'identifier', 'is_seekable', 'author', 'duration', 'stream', 'title', 'uri', 'requester',
-                 'preferences')
+                 'extra')
 
-    def __init__(self, requester, **kwargs):
+    def __init__(self, requester):
         self.requester = requester
-        self.preferences = kwargs
 
     @classmethod
-    def build(cls, track, requester, **kwargs):
+    def build(cls, track, requester, extra: dict = None):
```

Comment:
Isn't replacing kwargs with dict making things less readable? kwargs gets converted to dict anyway.

Reply:
I'm not sure, Devoxin wanted it to be a parameter that's a dict rather than kwargs

Reply:
@Devoxin pls explain

Reply:
My reasoning for the above is having stuff like `AudioTrack.build(..., spotify_playlist='blah', track_index=32)` is harder for people to interpret what the line does if looking at the code from an open-source perspective. Passing a dict instead like `AudioTrack.build(..., {'spotify_playlist': 'blah', 'track_index': 32})` looks clearer in portraying that it's a dictionary of extra information to associate with the track. Lastly, kwargs gets converted to a dict anyway as you say, which there's no point in wasting CPU cycles having the interpreter/compiler do this for us when the user is capable of doing it themselves.